entries/entry03.md

```diff
@@ -1,8 +1,27 @@
 # Entry 3
```

Comment:
Sources: 0 (need multiple, not using w3 twice)
EDP: 1
Knowledge: 0 (more code snippets!)
Skills: 1
Professionalism: 1